frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ unlockPassword }),
+      });
+
+      setDecryptedMnemonic(result?.mnemonic ?? "");
+      setIsDialogOpen(true);
+    } catch (error) {
+      toast({
+        title: "Incorrect password",
+        description: "Failed to decrypt mnemonic.",
+        variant: "destructive",
+      });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  async function onSubmit(e: React.FormEvent) {
+    e.preventDefault();
+
+    const sixMonthsLater = new Date();
+    sixMonthsLater.setMonth(sixMonthsLater.getMonth() + 6);
+
+    try {
+      await request("/api/backup-reminder", {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          nextBackupReminder: sixMonthsLater.toISOString(),
+        }),
+      });
+      await refetchInfo();
+      navigate("/");
+      toast({ title: "Recovery phrase backed up!" });
+    } catch (error) {
+      handleRequestError(toast, "Failed to store back up info", error);
+    }
+  }
+
+  return (
+    <>
+      <SettingsHeader
+        title="Backup"
+        description={
+          <>
+            <span className="text-muted-foreground">
+              Backup your wallet recovery phrase and or your channel states in
+              order to migrate your node.{" "}
+            </span>
+            <a
+              href="https://guides.getalby.com/user-guide/alby-account-and-browser-extension/alby-hub/backups-and-recover"
+              target="_blank"
+              rel="noreferrer noopener"
+              className="text-foreground underline"
+            >
+              {" "}
+              Learn more about backups
+            </a>
+          </>
+        }
+      />
+
+      {hasMnemonic && (
+        <div className="flex flex-col gap-6">
+          <div>
+            <h3 className="text-lg font-medium">Wallet Keys Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Key recovery phrase is a group of 12 random words that back up
+              your wallet on-chain balance. Using them is the only way to
+              recover access to your wallet on another machine or when you loose
+              your unlock password.
+            </p>
+          </div>
+          <p className="text-destructive">
+            If you loose access to your Hub and do not have your recovery
+            phrase, you will loose access to your funds.
+          </p>
+          {info?.backendType === "CASHU" && <CashuMnemonicWarning />}
+
+          <div>
+            <form
+              onSubmit={onSubmitPassword}
+              className="max-w-md flex flex-col gap-3"
+            >
+              <div className="grid gap-2 mb-6">
+                <Label htmlFor="password">Password</Label>
+                <PasswordInput id="password" onChange={setUnlockPassword} />
```

Comment:
minor: can this be `required` so when you click the button without entering password it shows you need to fill in this field?

Reply:
made it required at component level itself